src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,34 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.CountryDao;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
+import mate.academy.hibernate.relations.util.DataProcessingException;
 
 public class CountryServiceImpl implements CountryService {
+    private CountryDao countryDao;
+
+    public CountryServiceImpl(CountryDao countryDao) {
+        this.countryDao = countryDao;
+    }
+
     @Override
     public Country add(Country country) {
-        return null;
+        try {
+            return countryDao.add(country);
+        } catch (Exception e) {
+            throw new DataProcessingException("Error adding country: " + country, e);
+        }
     }
 
     @Override
     public Country get(Long id) {
-        return null;
+        try {
+            return countryDao.get(id)
```

Comment:
There is a missing semicolon at the end of the line. It should be `countryDao.get(id).orElseThrow(...);`